lighthouse-core/lib/asset-saver.js

```diff
@@ -54,19 +58,58 @@ img {
   `;
 }
 
+function loadArtifacts(basePath) {
+  log.log('Reading artifacts from disk:', basePath);
+  const artifacts = JSON.parse(fs.readFileSync(path.join(basePath, 'artifacts.json'), 'utf8'));
+  artifacts.traces = {};
+
+  const filenames = fs.readdirSync(basePath);
+  const promises = filenames.filter(filename => filename.endsWith('-trace.json')).map(filename => {
```

Comment:
should we do both traces and devtoolsLogs this way? It'd be nice to be consistent with how the assets are saved to disk in non-GAR mode but also less useful since there's nothing you can do with just the devtoolslog

nit: we also liked `.trace.json`/`.devtoolslog.json` if we want to stick with it here too :)